block-storage/src/main/scala/coop/rchain/blockstorage/errors.scala

```diff
@@ -1,58 +1,32 @@
 package coop.rchain.blockstorage
 
-import java.nio.file.Path
-
 import cats.data.EitherT
-import coop.rchain.casper.protocol.{BlockMessage, BlockMessageProto}
+import coop.rchain.casper.protocol.BlockMessage
 import coop.rchain.crypto.codec.Base16
 
 sealed abstract class StorageError extends Exception
 
-final case class CheckpointsDoNotStartFromZero(sortedCheckpoints: List[Path]) extends StorageError
-final case class CheckpointsAreNotConsecutive(sortedCheckpoints: List[Path])  extends StorageError
-final case class TopoSortLengthIsTooBig(length: Long)                         extends StorageError
+final case class TopoSortLengthIsTooBig(length: Long) extends StorageError
 final case class TopoSortFragmentParameterError(startBlockNumber: Long, endBlockNumber: Long)
     extends StorageError
 final case class BlockSenderIsMalformed(block: BlockMessage) extends StorageError
-final case class CheckpointDoesNotExist(offset: Long)        extends StorageError
-final case object LatestMessagesLogIsMalformed               extends StorageError
-final case object EquivocationsTrackerLogIsMalformed         extends StorageError
-final case object LatestMessagesLogIsCorrupted               extends StorageError
-final case object DataLookupIsCorrupted                      extends StorageError
-final case object InvalidBlocksIsCorrupted                   extends StorageError
-final case object BlockHashesByDeployLogIsCorrupted          extends StorageError
-final case object LastFinalizedBlockIsCorrupted              extends StorageError
+case object LatestMessagesLogIsCorrupted                     extends StorageError
+case object LastFinalizedBlockIsCorrupted                    extends StorageError
 
 object StorageError {
   type StorageErr[A]        = Either[StorageError, A]
   type StorageErrT[F[_], A] = EitherT[F, StorageError, A]
```

Comment:
This is also not used.

Reply:
This is still used in file block store isn't it?